### PR TITLE
Tanzu Application Platform  Deployment Guide updated for V 1.1

### PR DESCRIPTION
**Pull request by sendjainabhi**
Tanzu Application Platform  Deployment Guide updated for V 1.1
- Have changed TAP view/run/build/iterate profiles 
- Updated tap documentation version to 1.1

### CODE REVIEW

**Review by srinivasme21 — APPROVED** [review]
LGTM

File: src/deployment-guides/tap.md [hunk 1]
```diff
@@ -328,61 +280,28 @@ export TAP_REGISTRY_SERVER=<registry uri>
 export TAP_REGISTRY_USER=<registry user>
 export TAP_REGISTRY_PASSWORD=<registry password>
 export TAP_CNRS_DOMAIN=<cnrs domain>
+export alv_domain=< app live view app domain>
```
Comment (craigtracey) [inline src/deployment-guides/tap.md:283]: Please adjust all instances of `alv_domain` with `TAP_ALV_DOMAIN`. As these are exported environment variables they should be namespaced with `TAP_` prefix and it is customary to define environment variables in all-caps.

File: src/deployment-guides/tap.md [hunk 2]
```diff
@@ -328,61 +280,28 @@ export TAP_REGISTRY_SERVER=<registry uri>
...
- - learningcenter.tanzu.vmware.com
- - metadata-store.apps.tanzu.vmware.com
- - ootb-supply-chain-basic.tanzu.vmware.com
- - ootb-supply-chain-testing.tanzu.vmware.com
- - ootb-supply-chain-testing-scanning.tanzu.vmware.com
- - scanning.apps.tanzu.vmware.com
- - spring-boot-conventions.tanzu.vmware.com
- - tap-gui.tanzu.vmware.com
- - workshops.learningcenter.tanzu.vmware.com
+  domain_name: "${tap_run_cnrs_domain}"
```
Comment (craigtracey) [inline src/deployment-guides/tap.md:296]: this should be `domain_name: "${TAP_RUN_CNRS_DOMAIN}"`

File: src/deployment-guides/tap.md [hunk 3]
```diff
@@ -453,110 +479,57 @@ export TAP_REGISTRY_SERVER=<registry server uri>
 export TAP_REGISTRY_USER=<registry_user>
 export TAP_REGISTRY_PASSWORD=<registry_password>
 export TAP_GITHUB_TOKEN=<github_token>
-export TAP_APP_DOMAIN=<app_domain>
+export tap_view_app_domain=<sub domain of view cluster>
```
Comment (craigtracey) [inline src/deployment-guides/tap.md:482]: Same comment as above Please make this all-caps and adjust all references as such.

File: src/deployment-guides/tap.md [hunk 4]
```diff
@@ -609,71 +583,50 @@ export TAP_NAMESPACE="tap-install"
 export TAP_REGISTRY_SERVER=<registry server uri>
 export TAP_REGISTRY_USER=<registry user>
 export TAP_REGISTRY_PASSWORD=<registry password>
-export TAP_GITHUB_TOKEN=<github token>
-export TAP_APP_DOMAIN=<app domain>
-export TAP_GIT_CATALOG_URL=<git catalog url>
+export tap_iterate_cnrs_domain=<tap iterate cnrs  domain>
```
Comment (craigtracey) [inline src/deployment-guides/tap.md:586]: Please make all-caps and adjust references

File: src/deployment-guides/tap.md [hunk 5]
```diff
@@ -767,7 +721,7 @@ kubectl apply -f app-delivery.yaml
 kubectl get deliverables "${TAP_APP_NAME}"
 
 # get app url
-kubectl get all -A | grep route.serving.knative
+kubectl  get ksvc
```
Comment (craigtracey) [inline src/deployment-guides/tap.md:724]: Please remove extra space before `get`

**Review by craigtracey — CHANGES_REQUESTED** [review]

File: src/deployment-guides/tap.md (hunk 1 above)
Comment (sendjainabhi) [reply src/deployment-guides/tap.md]: done

File: src/deployment-guides/tap.md (hunk 2 above)
Comment (sendjainabhi) [reply src/deployment-guides/tap.md:296]: done

File: src/deployment-guides/tap.md [hunk 6]
```diff
@@ -328,61 +280,28 @@ export TAP_REGISTRY_SERVER=<registry uri>
...
- - ootb-supply-chain-testing-scanning.tanzu.vmware.com
- - scanning.apps.tanzu.vmware.com
- - spring-boot-conventions.tanzu.vmware.com
- - tap-gui.tanzu.vmware.com
- - workshops.learningcenter.tanzu.vmware.com
+  domain_name: "${tap_run_cnrs_domain}"
+appliveview_connector:
+  backend:
+    sslDisabled: true
+    host: appliveview.$alv_domain
```
[the remark this answers is not in the document]
Comment (sendjainabhi) [reply src/deployment-guides/tap.md]: done

File: src/deployment-guides/tap.md (hunk 3 above)
Comment (sendjainabhi) [reply src/deployment-guides/tap.md]: done

File: src/deployment-guides/tap.md (hunk 5 above)
Comment (sendjainabhi) [reply src/deployment-guides/tap.md]: done

**Review by craigtracey — APPROVED** [review]

**Pull request merged**